addOns/scripts/src/main/zapHomeFiles/scripts/templates/passive/Passive default template.js

```diff
@@ -3,7 +3,29 @@
 // Note that new passive scripts will initially be disabled
 // Right click the script in the Scripts tree and select "enable"  
 
-var PluginPassiveScanner = Java.type("org.zaproxy.zap.extension.pscan.PluginPassiveScanner");
+const PluginPassiveScanner = Java.type("org.zaproxy.zap.extension.pscan.PluginPassiveScanner");
```

Comment:
This might cause headaches for Java 11 users.

Reply:
Do you mean converting from `var` to `const`? I wasn't aware it did not work with Java 11, will change that back.

Reply:
Sorry for note being clear, not much about the Java version but the engine, Nashorn does not support const.